### ISSUE TITLE
Create a merge_cubes CLI

### ISSUE THREAD

**Issue by bayliffe** [metoppv/improver#295]
Prerequisite: IMPRO-331

As an IMPROVER developer I need a CLI for the merge_cubes utility so that I can easily include it when constructing suite apps.

Related issues: #284 

IMPROVER plugins are generally written to accept single input cubes which are then processed. When it comes to blending cycles and models we need to merge together disparate cubes using the merge_cubes utility before invoking the weighted-blending plugin. The CLI to be written in this work will enable us to invoke this merging in a consistent fashion from the relevant suite apps.

Acceptance criteria:
 * Create CLI for merge_cubes utility: `improver merge`  i.e. the plugins written in IMPRO-331/ #284 
 * Add CLI acceptance tests.
 * Updating existing apps is out of scope of this ticket.
